django-stubs/db/models/sql/query.pyi

```diff
@@ -111,7 +111,9 @@ class Query(BaseExpression):
     def get_initial_alias(self) -> str: ...
     def count_active_tables(self) -> int: ...
     def resolve_expression(self, query: Query, *args: Any, **kwargs: Any) -> Query: ...  # type: ignore[override]
-    def resolve_lookup_value(self, value: Any, can_reuse: set[str] | None, allow_joins: bool) -> Any: ...
+    def resolve_lookup_value(
+        self, value: Any, can_reuse: set[str] | None, allow_joins: bool, summarize: bool = ...
```

Comment:
Let's use literals in default vaues if they are simple.

```suggestion
        self, value: Any, can_reuse: set[str] | None, allow_joins: bool, summarize: bool = False
```

Reply:
I have a question. Is `Literal[False]` okay?

Reply:
No, the default value after `=` expects a *value*, not _the type of_ a literal value. You'd get a type error from mypy.

Reply:
Thank you for letting me know :)